utils.py

```diff
@@ -355,24 +358,24 @@ def write2canvas(imgs,spaces,indents,imgs_per_line):
 
             for count in range(no_of_words): 
                 if count in sdct: #Checking if space is required
-                    offset_w = offset_w + 20 
+                    offset_w = offset_w + space_in + random.randint(15,20)
                     continue
 
                 if count in idct: #Checking if indent is required
-                    offset_w = offset_w + 80
+                    offset_w = offset_w + indent_in + random.randint(15,20)
```

Comment:
if you do this we'll either have 100ish indents or tiny spaces coz space_in and indent_in are related